src/main/resources/application.properties

```diff
@@ -44,15 +44,14 @@ spring.mongodb.embedded.version: 3.4.1
 spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration
 
 # Details for connection to mongoDB
-spring.data.mongodb.host: localhost
-spring.data.mongodb.port: 27017
-#spring.data.mongodb.username:
-#spring.data.mongodb.password:
+# Configure uri Examples:
+# Example 1: mongodb://localhost:27017                                    - Single MongoDB instance, no authentication
+# Example 2: mongodb://username:mypasswd@localhost:27017                  - Single MongoDD instance with authentication
+# Example 3: mongodb://username:enc(apwf7Bc6mDuhvL)@localhost:27017       - Single MongoDD instance with encrypted password
```

Comment:
Do we actually use this uri with username and password in some test?

Reply:
Not yet.

Reply:
Now we do.